### ISSUE TITLE
[ERROR] Failed to execute goal on project aem-guides-wknd.core

### ISSUE THREAD

**Issue by kayest** [adobe/aem-guides-wknd#190]
I'm going through the "Project Setup" instructions here (https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/project-setup.html?lang=en#build) and got the following failure...

### Expected Behaviour
Pasting the following would result in a BUILD SUCCESS
```
mvn archetype:generate -B \
    -DarchetypeGroupId=com.adobe.granite.archetypes \
    -DarchetypeArtifactId=aem-project-archetype \
    -DarchetypeVersion=22 \
    -DgroupId=com.adobe.aem.guides \
    -Dversion=0.0.1-SNAPSHOT \
    -DappsFolderName=wknd \
    -DartifactId=aem-guides-wknd \
    -Dpackage=com.adobe.aem.guides.wknd \
    -DartifactName="WKND Sites Project" \
    -DcomponentGroupName=WKND \
    -DconfFolderName=wknd \
    -DcontentFolderName=wknd \
    -DcssId=wknd \
    -DisSingleCountryWebsite=n \
    -Dlanguage_country=en_us \
    -DoptionAemVersion=6.5.0 \
    -DoptionDispatcherConfig=none \
    -DoptionIncludeErrorHandler=n \
    -DoptionIncludeExamples=y \
    -DoptionIncludeFrontendModule=y \
    -DpackageGroup=wknd \
    -DsiteName="WKND Site" -X
```

### Actual Behaviour
This occurs instead:
```
[INFO] Reactor Summary for aem-guides-wknd 0.0.1-SNAPSHOT:
[INFO]
[INFO] aem-guides-wknd .................................... SUCCESS [  0.990 s]
[INFO] WKND Sites Project - Core .......................... FAILURE [01:32 min]
[INFO] WKND Sites Project - UI Frontend ................... SKIPPED
[INFO] WKND Sites Project - Repository Structure Package .. SKIPPED
[INFO] WKND Sites Project - UI apps ....................... SKIPPED
[INFO] WKND Sites Project - UI content .................... SKIPPED
[INFO] WKND Sites Project - All ........................... SKIPPED
[INFO] WKND Sites Project - Integration Tests Bundles ..... SKIPPED
[INFO] WKND Sites Project - Integration Tests Launcher .... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:37 min
[INFO] Finished at: 2021-01-30T12:49:20-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal biz.aQute.bnd:bnd-maven-plugin:4.2.0:bnd-process (bnd-process) on project aem-guides-wknd.core: bnd error: null: ConcurrentModificationException -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :aem-guides-wknd.core
```
### Reproduce Scenario (including but not limited to)
Starting over in a new directory causes the problem to reproduce. 

#### Steps to Reproduce
See above. 

#### Platform and Version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Program Files\apache-maven-3.6.3\bin\..
Java version: 15.0.1, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-15.0.1
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
AEM 6.5

**Comment by davidjgonzalez** [comment]
TLDR: use Java 11, not Java 15

This is due to your use of Java 15, which has an issue w this plugin. Regardless of this plug-ins issue, Java 15 is not supported by AEM so make sure you use Java 11 instead (you can download it on Adobe's software distribution site if you're an AEM customer).

**Comment by kayest** [comment]
Thanks. I'll research how to remove Java 15 next. 

When I go to App & Features on my Windows 10 machine, I noticed four JRE's running. Could that also contribute to the problem?

1. Java Runtime Environment 1.8.0_251
2. Java Runtime Environment 1.8.0_181
3. Java Runtime Environment 1.8.0_202
4. Java Runtime Environment 1.8.0_212

**Comment by kayest** [comment]
I installed Java 11, as advised above, and started the installation from scratch, in a new directory, carefully following every step here, https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/project-setup.html?lang=en#build, but am stilling getting that same error. Words can't describe the frustration after having tried to set up WKND on and off for three weeks now. 

 --- bnd-maven-plugin:4.2.0:bnd-process (bnd-process) @ aem-guides-wknd.core ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for aem-guides-wknd 0.0.1-SNAPSHOT:
[INFO]
[INFO] aem-guides-wknd .................................... SUCCESS [  0.942 s]
[INFO] WKND Sites Project - Core .......................... FAILURE [  5.259 s]
[INFO] WKND Sites Project - UI Frontend ................... SKIPPED
[INFO] WKND Sites Project - Repository Structure Package .. SKIPPED
[INFO] WKND Sites Project - UI apps ....................... SKIPPED
[INFO] WKND Sites Project - UI content .................... SKIPPED
[INFO] WKND Sites Project - All ........................... SKIPPED
[INFO] WKND Sites Project - Integration Tests Bundles ..... SKIPPED
[INFO] WKND Sites Project - Integration Tests Launcher .... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.801 s
[INFO] Finished at: 2021-01-30T17:33:00-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal biz.aQute.bnd:bnd-maven-plugin:4.2.0:bnd-process (bnd-process) on project aem-guides-wknd.core: bnd error: null: ConcurrentModificat
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command

**Comment by godanny86** [comment]
Hi @kayest, sorry for the frustration. Do you mind using the latest archetype version 25 to generate the project? We are about to release an updated tutorial that uses it on Monday.

1. `mvn --version`
  
  ```
  Apache Maven 3.6.2 (40f52333136460af0dc0d7232c0dc0bcf0d9e117; 2019-08-27T08:06:16-07:00)
  Maven home: /Library/apache-maven-3.6.2
  Java version: 11.0.4, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.4.jdk/Contents/Home
  Default locale: en_US, platform encoding: UTF-8
  ```

2. `java --version`

  ```
  java 11.0.4 2019-07-16 LTS
  Java(TM) SE Runtime Environment 18.9 (build 11.0.4+10-LTS)
  Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.4+10-LTS, mixed mode)
  ```

3. Generate the project (archetype 25), Assumes your target AEM environment is 6.5

  ```
  mvn -B archetype:generate \
        -D archetypeGroupId=com.adobe.aem \
        -D archetypeArtifactId=aem-project-archetype \
        -D archetypeVersion=25 \
        -D appTitle="WKND Sites Project" \
        -D appId="wknd" \
        -D groupId="com.adobe.aem.guides.wknd" \
        -D artifactId="aem-guides-wknd" \
        -D version="0.0.1-SNAPSHOT" \
        -D aemVersion="6.5"
  ```

4. Build and deploy

  ```
  $ cd aem-guides-wknd
  $ mvn clean install -PautoInstallSinglePackage
  ```

**Comment by kayest** [comment]
Thanks for the suggestion. I tried the above, except with Maven 3.6.3 and Java 11.0.10, but still get a failure at the same place:

[INFO] aem-guides-wknd .................................... SUCCESS [  0.870 s]
[INFO] WKND Sites Project - Core .......................... FAILURE [  4.814 s]
[INFO] WKND Sites Project - UI Frontend ................... SKIPPED
[INFO] WKND Sites Project - Repository Structure Package .. SKIPPED
[INFO] WKND Sites Project - UI apps ....................... SKIPPED
[INFO] WKND Sites Project - UI content .................... SKIPPED
[INFO] WKND Sites Project - All ........................... SKIPPED
[INFO] WKND Sites Project - Integration Tests Bundles ..... SKIPPED
[INFO] WKND Sites Project - Integration Tests Launcher .... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.759 s
[INFO] Finished at: 2021-01-30T19:08:20-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal biz.aQute.bnd:bnd-maven-plugin:4.2.0:bnd-process (bnd-process) on project aem-guides-wknd.core: bnd error: null: ConcurrentModificationException -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :aem-guides-wknd.core

Is there anything else I need to check on my PC, like Environment Variables?

**Comment by godanny86** [comment]
hmm, i thought the bnd version was updated to `5.1.2` in Archetype 24/25. Your error looks like it is `4.2.0`.

 In `aem-guides-wknd/pom.xml` what is the value of

```
<properties>
        <bnd.version>5.1.2</bnd.version>
```

**Comment by kayest** [comment]
You're right. It's 4.2.0. What's the best way to fix it... update that entry in the pom.xml file?

<bnd.version>4.2.0</bnd.version>

**Comment by godanny86** [comment]
Are you able to re-generate the project using Archetype 25?

```
mvn -B archetype:generate \
      -D archetypeGroupId=com.adobe.aem \
      -D archetypeArtifactId=aem-project-archetype \
      -D archetypeVersion=25 \
      -D appTitle="WKND Sites Project" \
      -D appId="wknd" \
      -D groupId="com.adobe.aem.guides.wknd" \
      -D artifactId="aem-guides-wknd" \
      -D version="0.0.1-SNAPSHOT" \
      -D aemVersion="6.5"
```

**Comment by kayest** [comment]
I'm not sure if I can resume the build, so I'll try this from square one in a new directory. I'll let you know how it goes. Thanks.

**Comment by kayest** [comment]
I tried to re-generate the project using the suggestion above and detailed every step below, however, I got another BUILD FAILURE. Here are the steps I took. I used Git Bash in Windows 10.

**STEP ONE**
Verified Maven and Java versions

$ mvn --version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: C:\Program Files\apache-maven-3.6.3
Java version: 11.0.10, vendor: Oracle Corporation, runtime: C:\Program Files\Java\jdk-11.0.10
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

**STEP TWO**
Referencing this page, https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/project-setup.html, I created new directory: wknd-test. Using Git Bash for everything below, I ran this in the parent folder and got BUILD SUCCESS:

mvn help:effective-settings

$ mvn help:effective-settings
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- maven-help-plugin:3.2.0:effective-settings (default-cli) @ standalone-pom ---
[INFO]
Effective user-specific configuration settings:

<?xml version="1.0" encoding="Cp1252"?>











<settings xmlns="http://maven.apache.org/SETTINGS/1.1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd">
  <localRepository>C:\Users\projects\.m2\repository</localRepository>
  <profiles>
    <profile>
      <activation>
        <activeByDefault>true</activeByDefault>
      </activation>
      <properties>
        <releaseRepository-Id>adobe-public-releases</releaseRepository-Id>
        <releaseRepository-Name>Adobe Public Releases</releaseRepository-Name>
        <releaseRepository-URL>https://repo.adobe.com/nexus/content/groups/public</releaseRepository-URL>
      </properties>
      <repositories>
        <repository>
          <releases>
            <updatePolicy>never</updatePolicy>
          </releases>
          <snapshots>
            <enabled>false</enabled>
          </snapshots>
          <id>adobe-public-releases</id>
          <name>Adobe Public Repository</name>
          <url>https://repo.adobe.com/nexus/content/groups/public</url>
        </repository>
      </repositories>
      <pluginRepositories>
        <pluginRepository>
          <releases>
            <updatePolicy>never</updatePolicy>
          </releases>
          <snapshots>
            <enabled>false</enabled>
          </snapshots>
          <id>adobe-public-releases</id>
          <name>Adobe Public Repository</name>
          <url>https://repo.adobe.com/nexus/content/groups/public</url>
        </pluginRepository>
      </pluginRepositories>
      <id>adobe-public</id>
    </profile>
  </profiles>
  <activeProfiles>
    <activeProfile>adobe-public</activeProfile>
  </activeProfiles>
  <pluginGroups>
    <pluginGroup>org.apache.maven.plugins</pluginGroup>
    <pluginGroup>org.codehaus.mojo</pluginGroup>
  </pluginGroups>
</settings>

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.879 s
[INFO] Finished at: 2021-01-31T17:05:47-05:00
[INFO] ------------------------------------------------------------------------

projects@GACDTL06projects MINGW64 ~

**STEP THREE**
Checked to see if adobe-public is present, which it is. 

**STEP FOUR**
Navigated to the **wknd-test** folder. 

**STEP FIVE**
Instead of pasting the following (which is posted on the aforementioned Adobe AEM page) into the command line to "generate the project in batch mode"...

mvn archetype:generate -B \
    -DarchetypeGroupId=com.adobe.granite.archetypes \
    -DarchetypeArtifactId=aem-project-archetype \
    -DarchetypeVersion=22 \
    -DgroupId=com.adobe.aem.guides \
    -Dversion=0.0.1-SNAPSHOT \
    -DappsFolderName=wknd \
    -DartifactId=aem-guides-wknd \
    -Dpackage=com.adobe.aem.guides.wknd \
    -DartifactName="WKND Sites Project" \
    -DcomponentGroupName=WKND \
    -DconfFolderName=wknd \
    -DcontentFolderName=wknd \
    -DcssId=wknd \
    -DisSingleCountryWebsite=n \
    -Dlanguage_country=en_us \
    -DoptionAemVersion=6.5.0 \
    -DoptionDispatcherConfig=none \
    -DoptionIncludeErrorHandler=n \
    -DoptionIncludeExamples=y \
    -DoptionIncludeFrontendModule=y \
    -DpackageGroup=wknd \
    -DsiteName="WKND Site"

I pasted this, which was recommended above:

mvn -B archetype:generate \
      -D archetypeGroupId=com.adobe.aem \
      -D archetypeArtifactId=aem-project-archetype \
      -D archetypeVersion=25 \
      -D appTitle="WKND Sites Project" \
      -D appId="wknd" \
      -D groupId="com.adobe.aem.guides.wknd" \
      -D artifactId="aem-guides-wknd" \
      -D version="0.0.1-SNAPSHOT" \
      -D aemVersion="6.5"

However, I got a BUILD FAILURE again. Here's the result:

projects@GACDTL06projects MINGW64 ~/wknd-test
$ mvn -B archetype:generate \
>       -D archetypeGroupId=com.adobe.aem \
>       -D archetypeArtifactId=aem-project-archetype \
>       -D archetypeVersion=25 \
>       -D appTitle="WKND Sites Project" \
>       -D appId="wknd" \
>       -D groupId="com.adobe.aem.guides.wknd" \
>       -D artifactId="aem-guides-wknd" \
>       -D version="0.0.1-SNAPSHOT" \
>       -D aemVersion="6.5"
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] >>> maven-archetype-plugin:3.2.0:generate (default-cli) > generate-sources @ standalone-pom >>>
[INFO]
[INFO] <<< maven-archetype-plugin:3.2.0:generate (default-cli) < generate-sources @ standalone-pom <<<
[INFO]
[INFO]
[INFO] --- maven-archetype-plugin:3.2.0:generate (default-cli) @ standalone-pom ---
[INFO] Generating project in Batch mode
[INFO] Archetype repository not defined. Using the one from [com.adobe.aem:aem-project-archetype:25] found in catalog remote
[INFO] ----------------------------------------------------------------------------
[INFO] Using following parameters for creating project from Archetype: aem-project-archetype:25
[INFO] ----------------------------------------------------------------------------
[INFO] Parameter: groupId, Value: com.adobe.aem.guides.wknd
[INFO] Parameter: artifactId, Value: aem-guides-wknd
[INFO] Parameter: version, Value: 0.0.1-SNAPSHOT
[INFO] Parameter: package, Value: com.adobe.aem.guides.wknd
[INFO] Parameter: packageInPathFormat, Value: com/adobe/aem/guides/wknd
[INFO] Parameter: includeForms, Value: n
[INFO] Parameter: country, Value: us
[INFO] Parameter: singleCountry, Value: y
[INFO] Parameter: includeErrorHandler, Value: n
[INFO] Parameter: datalayer, Value: y
[INFO] Parameter: includeDispatcherConfig, Value: y
[INFO] Parameter: sdkFormsVersion, Value: latest
[INFO] Parameter: package, Value: com.adobe.aem.guides.wknd
[INFO] Parameter: includeExamples, Value: n
[INFO] Parameter: amp, Value: n
[INFO] Parameter: groupId, Value: com.adobe.aem.guides.wknd
[INFO] Parameter: enableDynamicMedia, Value: n
[INFO] Parameter: language, Value: en
[INFO] Parameter: commerceEndpoint, Value: https://hostname.com/graphql
[INFO] Parameter: appTitle, Value: WKND Sites Project
[INFO] Parameter: version, Value: 0.0.1-SNAPSHOT
[INFO] Parameter: aemVersion, Value: 6.5
[INFO] Parameter: includeCommerce, Value: n
[INFO] Parameter: frontendModule, Value: general
[INFO] Parameter: appId, Value: wknd
[INFO] Parameter: artifactId, Value: aem-guides-wknd
[INFO] Parameter: sdkVersion, Value: latest
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\all\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\analyse\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\core\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\dispatcher.ams\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\dispatcher.cloud\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.frontend.general\pom.xml
[WARNING] Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.frontend.general\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.frontend.angular\pom.xml
[WARNING] Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.frontend.angular\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.frontend.react\pom.xml
[WARNING] Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.frontend.react\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.apps\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.config\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.content\pom.xml
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.content\src\main\content\jcr_root\content\wknd\_jcr_content\image\file
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\it.tests\pom.xml
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\pom.xml
[WARNING] Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\pom.xml
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\assets\image.png
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\wdio.conf.local.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\wdio.conf.commons.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\wdio.conf.cloud.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\specs\aem\sites.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\specs\aem\basic.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\specs\aem\assets.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\specs\aem\login.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\.eslintrc.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\lib\commons.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\lib\wdio.commands.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\lib\config.js
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\README.md
[WARNING] CP Don't override file C:\Users\projects\wknd-test\aem-guides-wknd\ui.tests\test-module\package.json
[INFO] Parent element not overwritten in C:\Users\projects\wknd-test\aem-guides-wknd\ui.apps.structure\pom.xml
[INFO] Executing META-INF/archetype-post-generate.groovy post-generation script
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass (file:/C:/Users/projects/.m2/repository/org/codehaus/groovy/groovy/2.4.16/groovy-2.4.16.jar) to method java.lang.Object.finalize()
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Creating content skeleton...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  19.766 s
[INFO] Finished at: 2021-01-31T17:15:49-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:3.2.0:generate (default-cli) on project standalone-pom: java.nio.file.FileSystemException: C:\Users\projects\wknd-test\aem-guides-wknd\dispatcher\src\conf.d\enabled_vhosts\aem_author.vhost: A required privilege is not held by the client. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

**Comment by kayest** [comment]
I got past the above blocker and proceeded to the next step, which is to navigate into the aem-guides-wknd project directory. I ran the following...

mvn -PautoInstallSinglePackage clean install

...but I got this again, at the same place as yesterday:

[INFO] Reactor Summary for aem-guides-wknd 0.0.1-SNAPSHOT:
[INFO]
[INFO] aem-guides-wknd .................................... SUCCESS [  9.452 s]
[INFO] WKND Sites Project - Core .......................... FAILURE [ 47.791 s]
[INFO] WKND Sites Project - UI Frontend ................... SKIPPED
[INFO] WKND Sites Project - Repository Structure Package .. SKIPPED
[INFO] WKND Sites Project - UI apps ....................... SKIPPED
[INFO] WKND Sites Project - UI content .................... SKIPPED
[INFO] WKND Sites Project - UI config ..................... SKIPPED
[INFO] WKND Sites Project - All ........................... SKIPPED
[INFO] WKND Sites Project - Integration Tests ............. SKIPPED
[INFO] WKND Sites Project - Dispatcher .................... SKIPPED
[INFO] WKND Sites Project - UI Tests ...................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:29 min
[INFO] Finished at: 2021-01-31T18:53:58-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project aem-guides-wknd.core: Could not resolve dependencies for project com.adobe.aem.guides.wknd:aem-guides-wknd.core:jar:0.0.1-SNAPSHOT: Could not find artifact com.adobe.aem:uber-jar:jar:apis:6.5 in adobe-public-releases (https://repo.adobe.com/nexus/content/groups/public) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :aem-guides-wknd.core

**Comment by godanny86** [comment]
hi @kayest my mistake. The snippet i gave you was slightly off, it should have been `6.5.0` instead of `6.5`:

```diff
mvn -B archetype:generate \
      -D archetypeGroupId=com.adobe.aem \
      -D archetypeArtifactId=aem-project-archetype \
      -D archetypeVersion=25 \
      -D appTitle="WKND Sites Project" \
      -D appId="wknd" \
      -D groupId="com.adobe.aem.guides.wknd" \
      -D artifactId="aem-guides-wknd" \
      -D version="0.0.1-SNAPSHOT" \
-     -D aemVersion="6.5"
+     -D aemVersion="6.5.0"
```

However, before I ask you to correct things further, what version of AEM are you targeting? AEM as a Cloud Service, AEM 6.5, AEM 6.4?

**Comment by kayest** [comment]
AEM 6.5

**Comment by godanny86** [comment]
Open `aem-guides-wknd/pom.xml` and modify the following dependency:

```diff
  <dependency>
        <groupId>com.adobe.aem</groupId>
        <artifactId>uber-jar</artifactId>
+      <version>6.5.0</version>
-      <version>6.5</version>
        <classifier>apis</classifier>
        <scope>provided</scope>
    </dependency>
```

**Comment by kayest** [comment]
Okay. I made the above modification.

**Comment by godanny86** [comment]
ok retry the Maven build from inside the `aem-guides-wknd` directory:

```
mvn -PautoInstallSinglePackage clean install
```

**Comment by kayest** [comment]
THANKS!!

[INFO] Reactor Summary for aem-guides-wknd 0.0.1-SNAPSHOT:
[INFO]
[INFO] aem-guides-wknd .................................... SUCCESS [  1.528 s]
[INFO] WKND Sites Project - Core .......................... SUCCESS [01:07 min]
[INFO] WKND Sites Project - UI Frontend ................... SUCCESS [03:31 min]
[INFO] WKND Sites Project - Repository Structure Package .. SUCCESS [  4.778 s]
[INFO] WKND Sites Project - UI apps ....................... SUCCESS [ 26.666 s]
[INFO] WKND Sites Project - UI content .................... SUCCESS [  9.939 s]
[INFO] WKND Sites Project - UI config ..................... SUCCESS [  0.986 s]
[INFO] WKND Sites Project - All ........................... SUCCESS [ 21.998 s]
[INFO] WKND Sites Project - Integration Tests ............. SUCCESS [03:33 min]
[INFO] WKND Sites Project - Dispatcher .................... SUCCESS [  1.435 s]
[INFO] WKND Sites Project - UI Tests ...................... SUCCESS [  4.408 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  09:30 min
[INFO] Finished at: 2021-01-31T19:43:34-05:00
[INFO] ------------------------------------------------------------------------

**Comment by godanny86** [comment]
Awesome! Updated tutorial should be published tomorrow. You'll be able to continue with the project you just created.

**Comment by kayest** [comment]
Great. Thanks for the help. Now I'll be resuming this: https://experienceleague.adobe.com/docs/experience-manager-learn/getting-started-wknd-tutorial-develop/project-setup.html?lang=en#project-structure.

**Comment by kayest** [comment]
The site still doesn't render properly at http://localhost:4502/content/wknd/language-masters/en.html

**Comment by godanny86** [comment]
Can you remove any previously deployed content from your AEM environment and re-deploy the project? The easiest way is from CRXDE Lite: http://localhost:4502/crx/de/index.jsp#/apps/wknd-packages  and delete `wknd-packages` under `/apps`. Otherwise manually deleting `/content/wknd`, `/content/experience-fragments/wknd`, `/conf/wknd`, and `/apps/wknd` via CRXDE lite and saving the changes should reset your environment.
